docs/users/overview.adoc

```diff
@@ -103,3 +103,31 @@ After navigating around, you may want to reset the zoom level and go back to a f
 image::images/bpmn-diagram_navigation_C.2.0.gif[BPMN Diagram Navigation]
 
 _Navigation with the https://github.com/bpmn-miwg/bpmn-miwg-test-suite/blob/cc75e467fd2b3009e67d4b24943591c66ce91a23/Reference/C.2.0.bpmn[migw-test-suite C.2.0] BPMN diagram_
+
+=== Purpose
+Process Analytics gives you the opportunity to visualize the Process Data on top of the BPMN diagram.
```

Comment:
In addition, I will add a link on `BPMN diagrams` that goes to the related paragraph of the page.